crates/env/src/engine/on_chain/ext/wasm32.rs

```diff
@@ -146,6 +146,15 @@ mod sys {
             output_ptr: Ptr32Mut<[u8]>,
         ) -> ReturnCode;
 
+        /// **WARNING**: this function is from the [unstable interface](https://github.com/paritytech/substrate/tree/master/frame/contracts#unstable-interfaces),
+        /// which is unsafe and normally is not available on production chains.
+        pub fn sr25519_verify(
```

Comment:
@athei @cmichi What do you think about adding a new `unstable` feature to the `ink_env` crate?